src/sgame/sg_bot_ai.cpp

```diff
@@ -325,6 +325,22 @@ Decorators
 Decorators are used to add functionality to the child node
 ======================
 */
+
+AINodeStatus_t BotDecoratorInvert( gentity_t *self, AIGenericNode_t *node )
+{
+	AIDecoratorNode_t *dec = ( AIDecoratorNode_t * ) node;
+
+	AINodeStatus_t status = BotEvaluateNode( self, dec->child );
+
+	if ( status == STATUS_SUCCESS )
```

Comment:
a switch seems to be what you really want here.

Reply:
maybe ? idk I like it like this